bin/generate_ci_matrix.py

```
@@ -60,31 +60,21 @@
-      # Always include board_level = 'pr'
-      if env["board_level"] == "pr":
-        outlist.append(env["ci"])
-      # Include board_level = 'extra' when requested
-      elif "extra" in args.level and env["board_level"] == "extra":
-        outlist.append(env["ci"])
-      # Include board_level = 'release' unless narrowed to the PR subset
-      elif "pr" not in args.level and env["board_level"] == "release":
-        outlist.append(env["ci"])
+# Filter builds by platform
+for env in all_envs:
+  if args.platform == env["ci"]["platform"] or args.platform == "all":
```

Flake8 (7.3.0)

[error] indentation is not a multiple of 4

(E111)